src/summaryComponent.js

```diff
@@ -9,13 +9,14 @@ function SummaryComponent(storyUrl) {
     var MIDDLEWARE_URL = "https://damp-headland-20696.herokuapp.com/aylien?apiRequestUrl="
     var AYLIEN_URL = "https://api.aylien.com/api/v1/summarize?url=";
 
-    var requestUrl = MIDDLEWARE_URL + AYLIEN_URL + storyUrl;
+    var requestUrl = `${MIDDLEWARE_URL} + ${AYLIEN_URL} + ${storyUrl}`;
```

Comment:
I might be wrong here, but does the url need "+" in there? now it's going to look like http://somethingsomething+somethingsomething+somestory.com. If so, then ignore this comment!